cmd/ausoceantv/src/checkout.js

```diff
@@ -0,0 +1,93 @@
+// This is your test publishable API key.
+const stripe = Stripe(import.meta.env.VITE_STRIPE_PUBLIC_KEY);
```

Comment:
This requires a .env file with the stripe public key

Reply:
Should this be committed? It is a public key, and will be different between development and production